<launch>

  <arg name="rviz" default="true" />

  <param name="offline" type="bool" value="false"/>
  <param name="pubcloud" type="bool" value="false"/>
  <param name="followpath" type="bool" value="true"/>
  
  <param name="objx" type="double" value="2.3"/>
  <param name="objy" type="double" value="3.0"/>
  <param name="objz" type="double" value="1.0"/>
  <param name="avoidcoff" type="double" value="1.44"/>
  <param name="filename" type="string" value="path"/>
  <param name="pathRadio" type="double" value="10.0"/>
  <param name="searchRadio" type="double" value="3.0"/>
  <param name="selfPoseInitX" type="double" value="0.0"/>
  <param name="selfPoseInitY" type="double" value="0.0"/>
  <param name="selfPoseInitT" type="double" value="0.0"/>
  <param name="idxDiff" type="double" value="1000.0"/>
  <param name="vehicleID" type="int" value="1"/>

  <node pkg="avoidance" type="extractObstacle" name="extractObstacle" output="screen" respawn="true">
    <!--remap from="/velodyne_points" to="/points_raw" /-->
  </node>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find avoidance)/rviz_cfg/avoidance.rviz" />
  </group>
</launch>
